package.json

```diff
@@ -14,28 +14,28 @@
     "start": "node packages",
     "dev": "nodemon packages",
     "migrate": "yarn server/db:migrate",
-
     "packages/build": "cd packages/guesstimator && yarn build && cd ../.. && cd packages/components && yarn build && cd ../.. && cd packages/prediction-analysis && yarn build && cd ../.. && cd packages/client && yarn build-t && cd ../..",
     "packages/clean": "cd packages/guesstimator && yarn clean && cd ../.. && cd packages/components && yarn clean && cd ../.. && cd packages/prediction-analysis && yarn clean && cd ../.. && cd packages/client && yarn clean && cd ../..",
     "packages/remove": "rm -rf ./packages/client/node_modules && rm -rf ./packages/components/node_modules && rm -rf ./packages/prediction-analysis/node_modules && rm -rf ./packages/guesstimator/node_modules",
-
     "server/db:create": "cd packages/server && yarn db:create && cd ../..",
     "server/db:migrate": "cd packages/server && yarn db:migrate cd ../..",
     "server/db:migrate:undo": "cd packages/server && yarn db:migrate:undo cd ../..",
     "server/db:reset": "cd packages/server && yarn db:reset && cd ../..",
     "server/db:seed": "cd packages/server && yarn db:seed && cd ../..",
     "server/test:ci": "cd packages/server && yarn test:ci && cd ../..",
     "server/lint:ci": "cd packages/server && yarn lint:ci && cd ../..",
-
     "google": "echo ${GOOGLE_AUTH} > ./packages/server/config/google.json",
-
     "preinstall": "printenv && yarn google",
-    "postinstall": "yarn bootstrap && yarn migrate && yarn packages/build && yarn packages/clean && yarn packages/remove"
+    "postinstall": "yarn bootstrap && yarn migrate && yarn packages/build && yarn packages/clean && yarn packages/remove",
+    "test": "mjs-mocha"
   },
```

Comment:
For what do we need "mjs-mocha" here?

Reply:
Try to merge "improvements/836" and than "cd packages/guesstimator && yarn test".

Reply:
I am trying to run the tests in foretold/packages/guesstimator/formatter/formatters/tests/lib-test.js